client/src/app/pages/applications/analysis-wizard/targets.ts

```diff
@@ -0,0 +1,198 @@
+
+export const targetsLabels = new Map([
+  ["eap7", "JBoss EAP 7"],
+  ["eap6", "JBoss EAP 6"],
+  ["azure-appservice", "Azure App Service"],
+  ["azure-aks", "Azure Kubernetes Service"],
+]);
+
+export const transformationTargets: ITransformationTargets[] = [
+  {
+    label: "Application server migration to",
+    description:
+      "Upgrade to the latest Release of JBoss EAP or migrate your applications to JBoss EAP from other Enterprise Application Server (e.g. Oracle WebLogic Server).",
+    options: ["eap7", "eap6"],
+    iconSrc: appOnServerIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Containerization",
+    description:
+      "A comprehensive set of cloud and container readiness rules to assess applications for suitability for deployment on Kubernetes.",
+    options: ["cloud-readiness"],
+    iconSrc: cloudIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Quarkus",
+    description:
+      "Rules to support the migration of Spring Boot applications to Quarkus.",
+    options: ["quarkus"],
+    iconSrc: migrationIcon,
+    forceSelect: false,
+  },
+  {
+    label: "OpenJDK",
+    description: "Rules to support the migration to OpenJDK from OracleJDK.",
+    options: ["openjdk"],
+    iconSrc: mugIcon,
+    forceSelect: false,
+  },
+  {
+    label: "OpenJDK 11",
+    description: "Rules to support the migration to OpenJDK 11 from OpenJDK 8",
+    options: ["openjdk11"],
+    iconSrc: mugIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Linux",
+    description:
+      "Ensure there are no Microsoft Windows paths hard coded into your applications.",
+    options: ["linux"],
+    iconSrc: serverIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Jakarta EE 9",
+    description:
+      "A collection of rules to support migrating applications from Java EE 8 to Jakarta EE 9. The rules cover project dependencies, package renaming, updating XML Schema namespaces, the renaming of application configuration properties and bootstraping files.",
+    options: ["jakarta-ee"],
+    iconSrc: migrationIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Spring Boot on Red Hat Runtimes",
+    description:
+      "A set of rules for assessing the compatibility of applications against the versions of Spring Boot libraries supported by Red Hat Runtimes.",
+    options: ["rhr"],
+    iconSrc: migrationIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Open Liberty",
+    description:
+      "A comprehensive set of rulesfor migrating traditional WebSphere applications to Open Liberty.",
+    options: ["openliberty"],
+    iconSrc: migrationIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Camel",
+    description:
+      "A comprehensive set of rules for migration from Apache Camel 2 to Apache Camel 3.",
+    options: ["camel"],
+    iconSrc: multiplyIcon,
+    forceSelect: false,
+  },
+  {
+    label: "Azure",
+    description:
+      "Upgrade your Java application so it can be deployed in different flavors of Azure.",
```

Comment:
@gildub Do we have any approved copy for this description? I just spotted one small grammar tweak. 

``` Upgrade your Java application so that it can be deployed in different flavors of Azure.```

Reply:
Just saw that this is how it is in the mockups. Disregard.